components/brave_wallet/common/features.cc

```diff
@@ -33,11 +33,7 @@ BASE_FEATURE(kBraveWalletBitcoinLedgerFeature,
 
 BASE_FEATURE(kBraveWalletZCashFeature,
              "BraveWalletZCash",
-#if BUILDFLAG(ENABLE_ZCASH_BY_DEFAULT)
```

Comment:
This build flag is currently set to:

```
enable_zcash_by_default = !is_ios && !is_android
```

You probably want to change this to:
```
enable_zcash_by_default = !is_android
```

If the intention is to enable it by default for iOS.

If the intention is to enable for android as well, then this GN arg should be removed entirely